### DISCUSSION TITLE
In+out this period on mobile?

### DISCUSSION THREAD

**Discussion by Plutoaurus** [firefly-iii/firefly-iii#7911] [Firefly III] [answered]
On the home screen on the desktop version there is the in+out, + spent figures.
On mobile, this is hidden however.

This figure alone has always been my best way of budgeting, other reports are for info.

Any way to get this on the mobile view without having to show desktop version that needs zooming and scrolling?

Thanks!

**Comment by JC5** [comment, marked as answer]
No, but the new layout I'm working on does have it!

**Reply by Plutoaurus** [reply]
Ooh exciting news.
Ta!
